egs/fame/v1/local/prepare_for_eer.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2015   David Snyder
+# Apache 2.0.
+#
+# Given a trials and scores file, this script 
+# prepares input for the binary compute-eer. 
+import sys
+trials = open(sys.argv[1], 'r').readlines()
+scores = open(sys.argv[2], 'r').readlines()
+spkrutt2target = {}
+for line in trials:
+  spkr, utt, target = line.strip().split()
+  spkrutt2target[spkr+utt]=target
+for line in scores:
+  spkr, utt, score = line.strip().split()
+  print score, spkrutt2target[spkr+utt]
```

Comment:
This script is probably redundant. In your run.sh, you could probably replace this with something like the following (but you'll have to verify that it works). 

`eer=$(paste <trials> <path-to-scores> | awk '{print $6, $3}' | compute-eer - 2>/dev/null)`